app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\RegisterRequest;
+use App\Models\Role;
+use App\Models\User;
+
+class RegisterController extends Controller
+{
+    public function register()
+    {
+        return view('cms.register');
+    }
+
+    public function handelRegister(RegisterRequest $request)
+    {
+        $user = new User();
+        $user->name = $request->name;
+        $user->email = $request->email;
+        $user->password = bcrypt($request->password);
+        $user->status = config('user.active');
+        $user->save();
+
+        $role = new Role();
+        $role->name = config('user.partner');
+        $role->description = 'thêm sửa xóa yêu cầu khách sạn';
+        $role->user_id = $user->id;
+        $role->save();
```

Comment:
sao thêm mới 1 user lại thêm mới 1 role ntn em?

Reply:
chỗ code này em làm đăng ký cho partner. quan hệ 1-1 bên bảng parter có user_id em để index

Reply:
như vậy ko đúng rồi. role với user là 1-n chứ. em chỉ cần gán cho user có role_id thôi. ntn thì bảng rolr của em toàn là data giống hệt nhau à?